assets/lang/sklmessages_pl_PL.properties

```diff
@@ -18,11 +18,11 @@ sidebar.profiles=Zainstalowane profile
 sidebar.loading=Wczytywanie profili...
 sidebar.settings=Ustawienia launchera
 sidebar.player=Gracz
-sidebar.guest=Witaj, nieznajomy! Proszę się zalogować.
+sidebar.guest=Witaj, nieznajomy! Proszę, zaloguj się.
```

Comment:
@CodeIndexer Co powiesz na samo 'zaloguj się'?

Reply:
Myślę, że już lepiej "Proszę, zaloguj się". "Zaloguj się" brzmi jak rozkaz (w tym przypadku). Wiadomo, że jest to wymagane, ale to od użytkownika zależy, czy będzie się chciał zalogować, czy nie. Nie chciałbym mu niczego nakazywać.
A tak swoją drogą, to czy ten tekst jest gdzieś używany? Chyba nigdzie się z nim nie spotkałem w UI launchera.

Reply:
Myślę, że już lepiej "Proszę, zaloguj się". "Zaloguj się" brzmi jak rozkaz (w tym przypadku). Wiadomo, że jest to wymagane, ale to od użytkownika zależy, czy będzie się chciał zalogować, czy nie. Nie chciałbym mu niczego nakazywać.
A tak swoją drogą, to czy ten tekst jest gdzieś używany? Chyba nigdzie się z nim nie spotkałem w UI launchera.

Reply:
Ja bym znowu nie przesadzał z grzecznościami. W końcu to launcher, a nie list. W Mojangowym launcherze jest podobnie

Reply:
Niby tak, ale to zdanie powinno się prawdopodobnie pojawiać przy pierwszym uruchomieniu launchera, a do nowych użytkowników, o których nie wiadomo, czy zostaną na dłużej, wolałbym raczej się zwracać z grzecznością. Ot tak po prostu. Ten tekst i tak będzie wyświetlał się (chyba) tylko raz, więc nie ma aż takiej potrzeby, żeby był on możliwie najkrótszy, kosztem grzeczności.

Reply:
Ok, masz rację. Co prawda przyznam że to dziwnie brzmi. Nie lepiej 'Proszę się zalogować'?

Reply:
Hmmm, racja. Niby "Proszę, zaloguj się" jest bliższe temu co w pliku źródłowym, ale po polsku chyba faktycznie lepiej brzmi "Proszę się zalogować". Nie wiem co w tej sytuacji zrobić :/

Reply:
@CodeIndexer Co o tym myślisz?

Reply:
Uważam, że lepiej brzmi "Proszę, zaloguj się".

Reply:
Ok, więc będzie "Proszę, zaloguj się".